metrics/README.md

```diff
@@ -1,67 +1,52 @@
 original src: https://github.com/docker/awesome-compose/tree/master/prometheus-grafana
 
-## Setup 
+## Setup
 
-requirements: 
-- `docker compose` 
-  - [https://docs.docker.com/engine/install/ubuntu/](https://docs.docker.com/engine/install/ubuntu/)
-- either mac or linux supported 
-
-modify `/etc/hosts` to include the following line: 
-```
-127.0.0.1 prometheus
-```
+requirements:
 
+- `docker compose`
+  - [https://docs.docker.com/engine/install/ubuntu/](https://docs.docker.com/engine/install/ubuntu/)
+- macos or linux (windows untested/unsupported)
 
-## Running 
+## Running
 
-mac: `docker compose -f compose_mac.yaml up -d`
-linux: `docker compose -f compose_linux.yaml up -d`
+mac: `docker compose up -d`
+linux: `docker compose up -d`
 
 - grafana will be accessable on `localhost:3000`
   - note: `username: admin password: grafana`
 - prometheus will be accessable on `localhost:9090`
-- sig metrics will be published to localhost:12345 (if you change this on the sig cli, you will 
-need to also modify the prometheus `target` to point to the different port).
+- sig metrics will be published to localhost:12345 (if you change this on the sig cli, you will
+  need to also modify the prometheus `target` to point to the different port).
+- sig gossip metrics are published to localhost:12355
```

Comment:
redundant with prior bullet

Reply:
addressed - messed up the commit on this one mb